modules/nested-elements/base/widget-nested-base.php

```diff
@@ -118,4 +118,15 @@ public function print_child( $index ) {
 			$children[ $index ]->print_element();
 		}
 	}
+
+	protected function content_template_single_repeater_item() {}
+
+	public function print_template() {
+		parent::print_template();
+		?>
+		<script type="text/html" id="tmpl-elementor-<?php echo esc_attr( $this->get_name() ); ?>-content-single">
+			<?php $this->content_template_single_repeater_item(); ?>
+		</script>
```

Comment:
@ArmanPurtoian You need to check if the widget supports it. Otherwise you might render a bunch of empty script tags.